src/hooks/useFormatGoalCalculatorConstants.ts

```diff
@@ -0,0 +1,70 @@
+import { useMemo } from 'react';
+import {
+  MpdGoalBenefitsConstantPlanEnum,
+  MpdGoalBenefitsConstantSizeEnum,
+  MpdGoalMiscConstantEnum,
+} from 'src/graphql/types.generated';
+import { GoalCalculatorConstantsQuery } from '../components/Reports/GoalCalculator/Shared/GoalCalculation.generated';
+
+export type GoalBenefitsConstantMap = Map<
+  string,
+  {
+    plan: MpdGoalBenefitsConstantPlanEnum;
+    planDisplayName: string;
+    size: MpdGoalBenefitsConstantSizeEnum;
+    sizeDisplayName: string;
+    cost: number;
+  }
+>;
+export type GoalMiscConstantMap = Map<MpdGoalMiscConstantEnum, string>;
+export type GoalGeographicConstantMap = Map<string, number>;
+
+interface FormattedConstants {
+  goalBenefitsConstantMap: GoalBenefitsConstantMap;
+  goalMiscConstantMap: GoalMiscConstantMap;
+  goalGeographicConstantMap: GoalGeographicConstantMap;
+}
+
+export const useFormatConstants = (
+  data?: GoalCalculatorConstantsQuery,
+): FormattedConstants => {
+  const goalBenefitsConstantMap: GoalBenefitsConstantMap = useMemo(() => {
```

Comment:
Since the map `key` doesn't appear to be used, I'm wondering what value this map adds? Could it be an array?

Reply:
We'll need this for the New Staff Calculator